plugins/stack-overflow-backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-stack-overflow-backend",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Note: we don't update the changelog manually, there is a guide [here](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets) explaining that if you create a changeset file, the changelog and also the version in the package.json file will be updated automatically 🙂.
Could you remove these changes, as you already created a changeset?